lib/testLauncher/buildArgs.js

```diff
@@ -12,6 +14,8 @@ const buildYargs = () => {
 		.boolean('help')
 		.boolean('version')
 		.help(false)
+		.check((hash, argv) => !argv.length && stColors.errorColor(texts['tl.noArguments']()), false)
+		.epilog(`Run ${stColors.bold('suitest interactive --help')} or ${stColors.bold('suitest automated --help')} to see those command's options`)
```

Comment:
Could you move the text to texts? There is a way to inject variables with tagged template.

Reply:
Also, I think there is no need for apostrophe in "command's" after all, as it's plural.